app/config/ezplatform.yml

```diff
@@ -57,10 +57,6 @@ ezpublish:
             # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
             http_cache:
                 purge_servers: ['%purge_server%']
-            # Temporary hard coding session name during v2 development
-            session:
-                name: eZSESSID
-            # ez-design-engine configuration
```

Comment:
should also drop `session` from admin_group

Reply:
Thanks. Updated in #70